src/Spryker/Configurator/DemoDateTimeConfiguratorPage/Theme/ConfiguratorApplication/src/services/types.ts

```diff
@@ -0,0 +1,29 @@
+export interface ProductData {
+    data?: Array<any>;
```

Comment:
Please use `...[]` for array notation
```suggestion
    data?: any[];
```